dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -762,6 +780,39 @@ public Limits setUserLimits(@ApiParam(hidden = true) @Auth User authUser,
         return getStrippedWorkflowsAndServices(userDAO.findById(user.getId()));
     }
 
+    @POST
```

Comment:
Channeling Charles, is this a PUT because it is idempotent?

Reply:
I'm not sure, it is idempotent, but it is just adding an entry to a join table.

Reply:
You are right, changing to PUT.

Reply:
hmm...might want to hold off on that

Reply:
@garyluu should it be a PUSH?

Reply:
With the current endpoint URL, it's unclear to me.  The way I see it,  REST practices interacts with "resources".  Take the "Users" resource for example.  If I were to completely replace a single user "/users/{userId}",  it'd be PUT.  If it's to create a new user, it'd be a POST "/users".  In this case, I personally think it might be a POST or a PATCH.  @coverbeck for input

Reply:
This is to update some data associated with an existing user. From Wikipedia "The PATCH method is a request method supported by the HTTP protocol for making partial changes to an existing resource." What about a using PATCH "/users/workflows"? @coverbeck  @garyluu